core-library/src/integration-test/java/org/silverpeas/core/admin/domain/SQLDomainServiceIT.java

```diff
@@ -208,11 +208,24 @@ public void testGetTechnicalDomainName() {
    */
   private void asserTechnicalDomainName(final String technicalDomainName, String domainId,
       String normalizedDomainName) {
-    String expectedTechnicalDomainName = StringUtil
-        .left(domainId + normalizedDomainName, SQLSettings.DATABASE_TABLE_NAME_MAX_LENGTH - 21);
+    String expectedTechnicalDomainName = left(domainId + normalizedDomainName,
```

Comment:
Because this method is only used once time and only in a test.
The goal to StringUtil isn't to provide all of the stuff of Apache Common StringUtils but to provide usual method actually used by our code and for this it wraps Apache Common StringUtils. By doing so, it could be more easy to move from an implementation to another one. The lesser we depend on a tiers code, the better we will be.

Reply:
OK, I was just wondering why the left method coded into this source, without any dependencies, was not introduced into StringUtil.